test/index.test.ts

```diff
@@ -401,3 +401,13 @@ it('resets cache when resetIsAvailableCache() is called', async () => {
   primary.done();
   assert.strictEqual(isGCE, false);
 });
+
+it('returns request timeout of 3000ms, when not GCF', () => {
+  assert.strictEqual(gcp.requestTimeout(), 3000);
+});
+
+it('returns request timeout of 500000ms, when in GCF environment', () => {
+  process.env.FUNCTION_NAME = 'my-function';
```

Comment:
Is it safe to assume this wouldn't override a testing environment's own variable?

Reply:
I should be, as I set the variable back post test (I've used this approach in the past and never bumped into too many issues).